lib/system/excpt.nim

```diff
@@ -140,8 +140,17 @@ proc closureIterSetupExc(e: ref Exception) {.compilerproc, inline.} =
 const
   nativeStackTraceSupported* = (defined(macosx) or defined(linux)) and
                               not NimStackTrace
-  hasSomeStackTrace = NimStackTrace or
-    defined(nativeStackTrace) and nativeStackTraceSupported
+  hasSomeStackTrace = NimStackTrace or defined(libbacktrace) or
+    (defined(nativeStackTrace) and nativeStackTraceSupported)
+
+when defined(libbacktrace):
+  # This is a Nim procedure, but we can't import the "libbacktrace" module in
+  # here, so we {.exportc.} it in there and {.importc.} it in here. We rely on
+  # "libbacktrace" being imported somewhere else, or this will cause a linker error.
+  proc getBacktrace*(): string {.importc.}
```

Comment:
exportc names are global so, again, should be prefixed to prevent potential clashes with user exportc code (including C code in this case!) => `nimGetBacktrace()`